<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ValArhivater - Архиватор папок</string>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="titleLabel">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>ValArhivater</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="archiveGroup">
      <property name="title">
       <string>Архивация папки</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="archiveButton">
           <property name="text">
            <string>Создать архив из папки</string>
           </property>
           <property name="icon">
            <iconset resource="icons/icons.qrc">
             <normaloff>:/icons/archive.png</normaloff>:/icons/archive.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="settingsButton">
           <property name="text">
            <string>Параметры</string>
           </property>
           <property name="icon">
            <iconset resource="icons/icons.qrc">
             <normaloff>:/icons/settings.png</normaloff>:/icons/settings.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="archiveStatusLabel">
         <property name="text">
          <string>Перетащите папку сюда или нажмите кнопку для выбора папки</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="extractGroup">
      <property name="title">
       <string>Распаковка архива</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="extractButton">
         <property name="text">
          <string>Распаковать архив</string>
         </property>
         <property name="icon">
          <iconset resource="icons/icons.qrc">
           <normaloff>:/icons/extract.png</normaloff>:/icons/extract.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="extractStatusLabel">
         <property name="text">
          <string>Перетащите архив сюда или нажмите кнопку для выбора архива</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <widget class="QLabel" name="statusLabel">
    <property name="text">
     <string>Поддерживаемые форматы: ZIP, TAR, GZ, RAR</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="icons/icons.qrc"/>
 </resources>
 <connections/>
</ui> 